pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -164,10 +166,12 @@ stdenv.mkDerivation rec {
     "-DARROW_BUILD_STATIC=${if enableShared then "OFF" else "ON"}"
     "-DARROW_BUILD_TESTS=ON"
     "-DARROW_BUILD_INTEGRATION=ON"
+    "-DTHRIFT_INCLUDE_DIR=${thrift}/include"
```

Comment:
This was necessary to prevent automatic discovery of `THRIFT_INCLUDE_DIR` as `${thrift}/include/thrift`. I'm not sure how this ever worked given that the invocation of `find_path` didn't change between 8.0.0 and 9.0.0.